conda_index/index/convert_cache.py

```diff
@@ -282,7 +258,7 @@ def convert_cache(conn, cache_generator):
                         },
                     )
 
-                else:
+                else:  # pragma: no cover
                     log.warn("Unhandled", match.groupdict())
```

Comment:
```suggestion
                    log.warn("Unhandled: %r", match.groupdict())
```

Without the `%r` - won't this ignore the second parameter?